stable_audio_tools/data/olac.py

```
@@ -0,0 +1,61 @@
+""" Tools to read and eventually write OLAC files. 
+"""
+import io
+import struct
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)